### ISSUE TITLE
MediaStream and HTMLMediaElement.captureStream()

### ISSUE THREAD

**Issue by luwes** [muxinc/media-tracks#4]
Discovered that Chrome and Firefox supports `HTMLMediaElement.captureStream()` and they return currently playing audio and video tracks. Should this project built it more around these API's?

https://developer.mozilla.org/en-US/docs/Web/API/MediaStream
https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/captureStream

--- 

Basically all browsers seem to have getters to get the currently playing tracks in some way. 

Safari supports HTMLMediaElement.videoTracks and HTMLMediaElement.audioTracks

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/audioTracks
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/videoTracks
